grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java

```diff
@@ -45,19 +50,43 @@
 
   private static final Logger logger = LoggerFactory.getLogger(BasicDaoImpl.class);
 
+  private static final String CONFIG_PROPERTY_FILE = "application.properties";
+
   private final JdbcTemplate jdbcTemplate;
 
+  private static long TIMESTAMP_RADIX = 1L;
+
   @Autowired
   public BasicDaoImpl(JdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
+    try (InputStream inputStream = new FileInputStream(new File(CONFIG_PROPERTY_FILE))) {
```

Comment:
Why use properties like that? https://www.baeldung.com/properties-with-spring

Reply:
I think this way is ok for this issue.